src/test/ui/did_you_mean/issue-38147-3.stderr

```diff
@@ -0,0 +1,13 @@
+error: cannot borrow immutable borrowed content `*self.s` as mutable
+  --> $DIR/issue-38147-3.rs:17:9
+   |
+12 |     s: &'a String
+   |     ------------- this field should be `&mut`
+...
+16 |     fn f(&self) {
+   |          ----- use `&mut self` here to make mutable
```

Comment:
I could look at the span's line number to identify order and change this to something like

```
error: cannot borrow immutable borrowed content `*self.s` as mutable
  --> $DIR/issue-38147-3.rs:17:9
   |
12 |     s: &'a String
   |     ------------- this field should be `&mut`...
...
16 |     fn f(&self) {
   |          ----- ...and use `&mut self` here to make mutable
17 |         self.s.push('x');
   |         ^^^^^^ cannot borrow as mutable
```

But I don't think it's unreasonable to show both labels, and if the developer only changes one of them, to still get the same error with the remaining label.